README.md

```diff
@@ -425,6 +429,24 @@ Furthermore, CTL exposes an `overlay` from its flake. You can use this in the Ni
 
 We have recenly set up a small scaffolding repository for projects wishing to adopt CTL: https://github.com/mlabs-haskell/ctl-scaffold. More documentation and resources will be added soon to the repo
 
+### Changing network configurations.
+
+CTL supports using networks other than testnet to provide development environment.
+
+First you need to specify an alternative way of getting the network configuration.
+
+[cardano-configurations](https://github.com/input-output-hk/cardano-configurations) repo provides configs for `mainnet`, `staging` and a few other networks.
+
+Which of the network config directories is chosen is determined by `network.name` parameter of `buildCtlRuntime`
+
+You can specify your own fork of `cardano-configurations` like this:
+
+```nix
```

Comment:
I'd remove `nix` here, since it's a fragment of something larger and not a valid Nix expression on its own, GH will render the markdown with highlighted syntax errors, like this:

```nix
foo = "...";
```